Prox/Prox/Models/Event.swift

```diff
@@ -202,8 +202,8 @@ class Event {
         return currentTime > lastNotificationTime - Event.eventStartNotificationInterval && currentTime < lastNotificationTime
     }
 
-    private func isFutureEvent(eventStartTime: Date, currentTime: Date) -> Bool {
```

Comment:
This method is called by: `internal func shouldShowEvent(withStartTime startTime: Date, endTime: Date?, ...`

It's fragile that `isFutureEvent` does not use the times passed in with `shouldShowEvent` (which is called with the instance's `startTime` and `endTime`).

Recommendations:
* (simpler, but perhaps misleading that it doesn't use instance vars) Revert `isFutureEvent` to take these parameters
* Remove the startTime/endTime parameters from `shouldShowEvent`. I think they're there to make for easier testing so you'd have to add a test helper method like `getEvent(withStartTime:withEndTime) -> Event` or something and fix up the tests.